oc_dms_mirror/rest_api/app/__init__.py

```
@@ -1,12 +1,14 @@
 from flask import Flask, Blueprint
 
-dms_mirror_bp = Blueprint("dms_mirror_bp", __name__)
 from .routes import *
 
+def register_blueprint_controller(app):
+    dms_mirror_bluprint = DmsMirrorBlueprint()
```

Ruff (0.14.2)

`DmsMirrorBlueprint` may be undefined, or defined from star imports

(F405)